### ISSUE TITLE
Columns Block: Video block does not resize to column width

### ISSUE THREAD

**Issue by joweber123** [Automattic/wp-calypso#42871]
### Steps to reproduce

1. Add a column block to the page.
2. Inside of this column block add a video block.

#### What I expected
I expected the video block to resize to the width of the column.

#### What happened instead
The video block is cutting off the video and displaying full height.

#### Browser / OS version
Firefox/Mac OS Catalina

#### Screenshot / Video
![video_block](https://user-images.githubusercontent.com/24914028/83478954-25a6d780-a4d2-11ea-9735-3afc51e941b8.png)

#### Context / Source
3021461-zen

**Comment by simison** [comment]
cc @lessbloat would this be for Manage to triage?

**Comment by joweber123** [comment]
> Which theme were they using? 

I have tested on the Varia child-themes and this seems to occur on all of them.  The original report was on an older theme Veni.  On these themes, the issue occurs in both the editor as well as when viewing the site live. 

On Twenty-twenty it does appear the theme resizes the videos when viewing the site.  In the editor though, the videos are stretched regardless of theme.

### Editor
![column_video_bug](https://user-images.githubusercontent.com/24914028/83505392-b1375d00-a500-11ea-8536-81075eec0b3c.png)

### Front-end Varia child-theme
![front_end_video_bug](https://user-images.githubusercontent.com/24914028/83505729-2acf4b00-a501-11ea-9c9d-b2e0d6d183b7.png)

### Front-end Twenty-twenty
![video_bug_site](https://user-images.githubusercontent.com/24914028/83505512-db891a80-a500-11ea-9cbb-d47ecdb16318.png)


@simison

**Comment by jordesign** [comment]
I've got another couple of examples of this - and seems to be happening even in internal P2s. 
Wondering if we can review this @davemart-in ?

**Comment by davemart-in** [comment]
Yep! Thanks for the ping. Moved this to our prioritized janitor backlog. 👍
